Marlin/ultralcd_impl_DOGM.h

```diff
@@ -318,10 +318,10 @@ FORCE_INLINE void _draw_heater_status(const uint8_t x, const int8_t heater) {
   #endif
 
   if (PAGE_UNDER(7))
-    _draw_centered_temp((isBed ? thermalManager.degTargetBed() : thermalManager.degTargetHotend(heater)) + 0.5, x, 7);
+    _draw_centered_temp((isBed ? thermalManager.degTargetBed() : thermalManager.degTargetHotend(heater)) + 0.5, x, 7-1);
 
   if (PAGE_CONTAINS(21, 28))
-    _draw_centered_temp((isBed ? thermalManager.degBed() : thermalManager.degHotend(heater)) + 0.5, x, 28);
+    _draw_centered_temp((isBed ? thermalManager.degBed() : thermalManager.degHotend(heater)) + 0.5, x, 28-1);
```

Comment:
@yhfudev Is it possible here for the glyph to extend below pixel row 27 and into pixel row 28?

I ask, because if the position of the glyph changes we should alter `PAGE_CONTAINS(21, 28)` too.